tests/test_bare.sh

```diff
@@ -18,7 +18,7 @@ mkdir -p .cache/bare
 cd .cache/bare
 
 # create the project using the default settings in cookiecutter.json
-cookiecutter ../../ --no-input --overwrite-if-exists use_docker=n $@
+cookiecutter ../../ --no-input --overwrite-if-exists use_docker=n use_async=y $@
```

Comment:
Can we remove this change? If you want to test this on CI, I'd rather add it to the `travis.yml`. This script is just to test non-docker, all the other arguments should be passed by the caller.

Reply:
Sure!